session/src/main/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandler.java

```diff
@@ -123,9 +124,20 @@ public APIGatewayProxyResponseEvent handleRequest(
             auditSessionItem.setSubject(sessionRequest.getSubject());
             auditSessionItem.setPersistentSessionId(sessionRequest.getPersistentSessionId());
             auditSessionItem.setClientSessionId(sessionRequest.getClientSessionId());
-            auditService.sendAuditEvent(
-                    AuditEventType.START,
-                    new AuditEventContext(input.getHeaders(), auditSessionItem));
+            if (!StringUtils.isBlank(sessionRequest.getContext())) {
+                List<Evidence> evidenceList = new ArrayList<>();
+                Evidence evidence = new Evidence();
+                evidence.setContext(sessionRequest.getContext());
+                evidenceList.add(evidence);
+                auditService.sendAuditEvent(
+                        AuditEventType.START,
+                        new AuditEventContext(input.getHeaders(), auditSessionItem),
+                        new AuditEventExtensions(evidenceList));
+            } else {
+                auditService.sendAuditEvent(
+                        AuditEventType.START,
+                        new AuditEventContext(input.getHeaders(), auditSessionItem));
+            }
```

Comment:
Maybe we could extract this into a method something like 

```
public void sendStartAuditEvent(SessionRequest sessionRequest) {
 List<Evidence> evidenceList = new ArrayList<>();
        if (!StringUtils.isBlank(sessionRequest.getContext())) {
            Evidence evidence = new Evidence();
            evidence.setContext(sessionRequest.getContext());
            evidenceList.add(evidence);
        }

        auditService.sendAuditEvent(
            AuditEventType.START,
            new AuditEventContext(headers, auditSessionItem),
            evidenceList.isEmpty() ? null : new AuditEventExtensions(evidenceList)
        );
}
```

Reply:
Extracted method outside of handle request